Mathlib/ModelTheory/Substructures.lean

```diff
@@ -885,8 +887,295 @@ theorem range_subtype (S : L.Substructure M) : S.subtype.toHom.range = S := by
   rintro ⟨⟨y, hy⟩, rfl⟩
   exact hy
 
+variable (L) (M) (N)
+
+/-- An equivalence between substructures -/
+structure SubEquivalence  where
+  /--The substructure which is the domain of the equivalence.-/
+  sub_dom : L.Substructure M
+  /--The substructure which is the codomain of the equivalence.-/
+  sub_cod : L.Substructure N
+  /--The equivalence between the two subdomains.-/
```

Comment:
For clarity, almost all files have the spaces here. I don't particularly mind either way, but now this PR is making this file stand out

Reply:
Looks you went for restoring the spaces but only restored the leading ones, not the trailing ones?

Reply:
Oops - another find and replace mistake.
I think I've fixed that now, but I think I should make a separate PR to go and clean up all the whitespace in the model theory files, since that isn't the point of this PR, and I see some other whitespace issues in my files.